Enigmatry.Entry.Infrastructure.Tests/TimeProviderFixture.cs

```diff
@@ -7,22 +7,24 @@ namespace Enigmatry.Entry.Infrastructure.Tests;
 public class TimeProviderFixture
 {
     [Test]
-    public void SameTime()
+    public async Task SameTime()
     {
         var provider = new TimeProvider();
 
         var firstDate = provider.FixedUtcNow;
+        await Task.Delay(TimeSpan.FromMilliseconds(1));
```

Comment:
Only minor, optional change to use Thread.Sleep instead of Task.Delay. No need for async in this test.

Reply:
Updated tests to use Thread.Sleep